libraries/protocol/include/steem/protocol/asset.hpp

```diff
@@ -85,6 +85,22 @@ namespace steem { namespace protocol {
       }
    };
 
+   /** Helper comparator facilitating asset (or asset_symbol_type)
+    *  storage sorted by symbol's nai.
+    */
+   struct nai_less
```

Comment:
This comparator is not needed since `add_reward_token` is not needed.

If we want to abstract anything to be re-usable, it would be the validation of `vector< asset >` done in `claim_reward_balance2_operation`.

Reply:
Removed.